docs/how-to/build-wallets-signing-tools-api-clients.md

```diff
@@ -9,19 +9,21 @@ description: Additional building blocks for Filecoin applications
 
 To simplify or automate how users make payments, you can integrate one or more of these wallets directly into your application or service:
 
-**_TODO: Add wallet options and descriptions when ready_**
+- [Ledger integration library](https://github.com/Zondax/ledger-filecoin/) - for developers of wallets or applications wishing to integrate with the Ledger hardware device
+- [Glif web wallet](https://github.com/openworklabs/filecoin-web-wallet) (in progress) - web wallet built with Next.js that supports sending & receiving FIL from a Ledger device
+- Mobile and other wallet integrations (to come)
 
 
 ## Signing Libaries
```

Comment:
Just confirming you intended to leave this as a level 2 header and have it surfaced in the nav?

Reply:
Yes for this situation — because it's a specific reference that a dev might look up multiple times while designing/setting up an application.